bindgen/clang.rs

```diff
@@ -2351,7 +2351,7 @@ impl EvalResult {
 
         if unsafe { clang_EvalResult_isUnsignedInt(self.x) } != 0 {
             let value = unsafe { clang_EvalResult_getAsUnsigned(self.x) };
-            if value > i64::MAX as c_ulonglong {
+            if value > u64::MAX as c_ulonglong {
```

Comment:
I'm a bit surprised that this doesn't trigger a warning for case where c_ulonglong is `u64` but alright.